packages/dmn-js-decision-table/assets/css/dmn-js-decision-table.css

```diff
@@ -183,18 +183,28 @@
 }
 
 /* decision table properties */
+.dmn-decision-table-container .decision-table-properties {
+  display: flex;

```

Comment:
Are we introducing both flexbox and CSS grid with this PR?

Reply:
Indeed, CSS Grid is introduced with this PR.

Flexbox has been there for a while, though: https://github.com/bpmn-io/dmn-js/search?q=display%3A+flex&unscoped_q=display%3A+flex

Reply:
I think it's okay then to have it both unless we have no other chance to use flexbox for the same purpose for which we introduce CSS Grid.

Reply:
To be honest, I don't really see a way to implement new layout with flexbox instead. It would be especially difficult or event impossible to position the logo visually inside of the decision table properties and maintain independence between two modules. We explicitly do not want to make PoweredByModule depend on the decision tabel properties, as discussed with @philippfromme.

Also, Flexbox and CSS Grid are just two different ways of layouting with different purposes. We do 2D layouting in this PR and in my opinion CSS Grid is of perfect use here. For my motivations, see also: https://www.youtube.com/watch?v=hs3piaN4b5I

Reply:
Also, @philippfromme and I tried to run this on IE11 and discovered that changes introduced 3 months ago have already broken support for that browser. I've prepared IE11 compat commit, though: https://github.com/bpmn-io/dmn-js/commit/57d4c402c0d5d2540fbceba4a5da2951a827bdc8

Reply:
Thanks for the explanation! 👍